web/template/layout/mlpa-header.gohtml

```diff
@@ -62,8 +62,8 @@
             <nav data-module="dropdown-menu">
                 <select class="govuk-select" id="case-actions" data-role="dropdown-menu-toggle">
                     <option value="case-actions" selected>Case actions</option>
-                    <option value="{{ prefix (printf "/create-task?id=%d" .CaseSummary.Lpa.ID) }}">Create a task</option>
-                    <option value="{{ prefix (printf "/create-warning?id=%d" .CaseSummary.Lpa.Donor.ID) }}">Create a warning</option>
+                    <option value="{{ prefix (printf "/create-task?id=%d" .CaseSummary.DigitalLpa.SiriusData.ID) }}">Create a task</option>
+                    <option value="{{ prefix (printf "/create-warning?id=%d" .CaseSummary.DigitalLpa.SiriusData.ID) }}">Create a warning</option>
```

Comment:
We should be creating warnings on the case rather than the donor now so I corrected this